tests/test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -x
```

Comment:
+1

suggest using `-x` until someone makes the output better as you suggested. Would rather have `-x` than the current state of no output.